packages/0xcert-ethereum-erc721-contracts/src/contracts/nf-token-metadata-enumerable.sol

```diff
@@ -132,7 +137,8 @@ contract NFTokenMetadataEnumerable is
 
   /**
    * @dev Contract constructor.
-   * @notice When implementing this contract don't forget to set nftName, nftSymbol and uriBase.
+   * @notice When implementing this contract don't forget to set nftName, nftSymbol, uriPrefix and
```

Comment:
* @notice When implementing this contract, don't forget to set nftName, nftSymbol, uriPrefix and